File: ORB.Core/Models/CorporateRiskMitigationActionUpdate.cs
using System;
using System.Collections.Generic;

namespace ORB.Core.Models
{
    public partial class CorporateRiskMitigationActionUpdate : RiskMitigationActionUpdate
    {
        public virtual CorporateRiskMitigationAction RiskMitigationAction { get; set; }
        public virtual CorporateRiskUpdate RiskUpdate { get; set; }
    }
}

File: corporate-reporting/src/types/CorporateRiskMitigationActionUpdate.ts
import { IRiskMitigationActionUpdate, RiskMitigationActionUpdate } from "./RiskMitigationActionUpdate";
import { ICorporateRiskMitigationAction } from "./CorporateRiskMitigationAction";

export interface ICorporateRiskMitigationActionUpdate extends IRiskMitigationActionUpdate {
    RiskMitigationActionID: number;
    SignOffID: number;
    RiskMitigationAction?: ICorporateRiskMitigationAction;
}

export class CorporateRiskMitigationActionUpdate extends RiskMitigationActionUpdate implements ICorporateRiskMitigationActionUpdate { }